src/page/categoryPage/CategoryPage.jsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { quizeCategories } from "../../backend/db/quizCategory";
+import { Navbar } from "../../components";
+import "./category-page.css";
+// import categoryPageStyles from "./category-page.modules.css";
+
+export default function CategoryPage() {
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    setCategories((categories) => quizeCategories.categories);
+  }, []);
+
+  return (
+    <section className="category-page-namespace page-wrap">
+      <section className="category-page-namespace page-nav">
+        <Navbar></Navbar>
```

Comment:
suggestion: you can use self closing tag here like this: `<Navbar />`

Reply:
Done, in the upcoming PR